.github/workflows/publish-code.yml

```diff
@@ -72,3 +72,26 @@ jobs:
       - name: Publish Nuget packages to Nuget registry
         working-directory: src
         run: dotnet nuget push "out/*" -k ${{secrets.NUGET_AUTH_TOKEN}}
+      - name: Zip documentation
+        working-directory: docs2/public
+        run: zip -r ../../src/out/Documentation.zip "./*"
```

Comment:
Is `zip -r ../../src/out/Documentation.zip "./*"` correct?  I added the quotes so we don't have problems with bash again.

Reply:
See https://stackoverflow.com/questions/26646616/how-to-create-a-zip-file-using-shell-script

Reply:
OK